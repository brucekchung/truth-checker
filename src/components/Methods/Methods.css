.Methods {
  background: white;
  width: 95%;
  margin: 0 auto;
  margin-top: 15px;
  border-radius: 14px;
  padding: 15px 0 15px 0;
}