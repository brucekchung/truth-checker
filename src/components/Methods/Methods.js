import React from 'react'
import './Methods.css'

export const Methods = () => (
  <div className="Methods">
    <h1>Methods</h1>
    <p>Categories of Evaluation: </p>
    <p>Organization, Author, and Content</p>
    <p>
      Organization - uses the Better Business Bureau (BBB) api. 
      A letter rating A - F is parsed from the data.
    </p>
    <p>
      Author - uses GoogleScholar via node.js module.
      Author data returned includes length of search.
      Citations based of the Author&apos;s works are currently not functioning in the node module.
    </p>
    <p>
      Content - uses the Watson Tone Analyzer api.
      Information parsed includes anger and fear (strong indicators of a biased article)
      and &apos;analytical&apos; - an emotional tone indicating potential for a balanced article.
      Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively.
      Content analysis does not attempt to fact-check.
    </p>
  </div>
)