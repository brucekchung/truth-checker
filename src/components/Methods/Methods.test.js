import React from 'react'
import { Methods } from './Methods'
import { shallow } from 'enzyme'

describe('Methods', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Methods />)

    expect(wrapper).toMatchSnapshot()
  })
})