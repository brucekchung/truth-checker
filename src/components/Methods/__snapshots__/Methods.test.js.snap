// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Methods should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Methods
</h1>,
        <p>
          Categories of Evaluation: 
</p>,
        <p>
          Organization, Author, and Content
</p>,
        <p>
          Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.
</p>,
        <p>
          Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.
</p>,
        <p>
          Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Methods",
        },
        "ref": null,
        "rendered": "Methods",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Categories of Evaluation: ",
        },
        "ref": null,
        "rendered": "Categories of Evaluation: ",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Organization, Author, and Content",
        },
        "ref": null,
        "rendered": "Organization, Author, and Content",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.",
        },
        "ref": null,
        "rendered": "Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.",
        },
        "ref": null,
        "rendered": "Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.",
        },
        "ref": null,
        "rendered": "Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Methods
</h1>,
          <p>
            Categories of Evaluation: 
</p>,
          <p>
            Organization, Author, and Content
</p>,
          <p>
            Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.
</p>,
          <p>
            Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.
</p>,
          <p>
            Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Methods",
          },
          "ref": null,
          "rendered": "Methods",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Categories of Evaluation: ",
          },
          "ref": null,
          "rendered": "Categories of Evaluation: ",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Organization, Author, and Content",
          },
          "ref": null,
          "rendered": "Organization, Author, and Content",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.",
          },
          "ref": null,
          "rendered": "Organization - uses the Better Business Bureau (BBB) api. A letter rating A - F is parsed from the data.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.",
          },
          "ref": null,
          "rendered": "Author - uses GoogleScholar via node.js module. Author data returned includes length of search. Citations based of the Author's works are currently not functioning in the node module.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.",
          },
          "ref": null,
          "rendered": "Content - uses the Watson Tone Analyzer api. Information parsed includes anger and fear (strong indicators of a biased article) and 'analytical' - an emotional tone indicating potential for a balanced article. Content that scores low in anger/fear and high in analytical is assessed positively, and the reverse negatively. Content analysis does not attempt to fact-check.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
